### PR TITLE
Next

### PR DESCRIPTION
**Pull request by Sitronik**
Added vue 3 support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,43 +1,45 @@
 {
-  "name": "vue-element-loading",
+  "name": "vue3-element-loading",
```
Comment (dacoto97) [inline package.json:2]: Set original name

File: package.json [hunk 2]
```diff
@@ -1,43 +1,45 @@
...
   "description": "Loading inside a container or full screen for Vue.js",
-  "version": "1.1.6",
-  "author": "biig_pongsatorn <biig_pongsatorn@hotmail.com>",
+  "version": "1.1.7",
```
Comment (dacoto97) [inline package.json:4]: Mayor update, change versión to 2.0.0

File: package.json [hunk 3]
```diff
@@ -1,43 +1,45 @@
...
+  "author": "sitronik",
```
Comment (dacoto97) [inline package.json:5]: Incorrect original author: biig_pongsatorn <biig_pongsatorn@hotmail.com>

File: package.json [hunk 4]
```diff
@@ -1,43 +1,45 @@
...
-  "main": "lib/vue-element-loading.min.js",
+  "main": "dist/VueElementLoading.common.js",
   "files": [
-    "lib"
+    "dist"
   ],
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/biigpongsatorn/vue-element-loading.git"
+    "url": "git+https://github.com/Sitronik/vue-element-loading.git"
```
Comment (dacoto97) [inline package.json:13]: Incorrect URL: git+https://github.com/biigpongsatorn/vue-element-loading.git

**Review by dacoto97 — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (Sitronik) [reply package.json]: Ok

File: package.json (hunk 3 above)
Comment (Sitronik) [reply package.json]: Ok

File: package.json (hunk 2 above)
Comment (Sitronik) [reply package.json]: Ok

File: package.json (hunk 1 above)
Comment (Sitronik) [reply package.json]: Ok

**Comment by Sitronik** [comment]
@dacoto97  I changed this to publish a temporary package to npm. I changed it now as it was before

**Review by dacoto97 — APPROVED** [review]

**Comment by runyasak** [comment]
@Sitronik 
`Toggle (Pikachu)` button is working but Pikachu gif doesn't appear.
Please, fix my adorable Pikachu to be appeared. 😂

**Comment by Sitronik** [comment]
> @Sitronik
> `Toggle (Pikachu)` button is working but Pikachu gif doesn't appear.
> Please, fix my adorable Pikachu to be appeared. 😂

Fixed c95820d

**Comment by runyasak** [comment]
image

@Sitronik There have some warning about `vue-loader` version.
Could you please fix this warning ?

**Comment by Sitronik** [comment]
> image
> 
> @Sitronik There have some warning about `vue-loader` version.
> Could you please fix this warning ?

fixed 6dbcc7b

**Review by runyasak — APPROVED** [review]
That's awesome. 😂 🎉
Thank you for your contributing. 😁

File: package.json [hunk 5]
```diff
@@ -1,43 +1,45 @@
...
+    "dist"
   ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/biigpongsatorn/vue-element-loading.git"
   },
   "scripts": {
     "dev": "cross-env NODE_ENV=development webpack-dev-server --mode development --open --hot",
-    "build:lib": "rm -rf ./lib && bili src/index.vue",
+    "build": "npx vue-cli-service build --target lib --name VueElementLoading ./src/index.js --mode production",
```
[the remark this answers is not in the document]
Comment (Sitronik) [reply package.json:17]: ok

**Pull request merged**